MAUI/DataGrid/scrolling.md

```diff
@@ -48,4 +48,41 @@ You can scroll programmatically to a particular column using the `SfDataGrid.Scr
 {% highlight C# %}
 this.dataGrid.ScrollToColumnIndex(4, true, ScrollToPosition.MakeVisible);
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+## Scrollbar Visibility
+
+You can change the visibility of the horizontal and vertical scrollbar using the [SfDataGrid.HorizontalScrollBarVisibility]() and [SfDataGrid.VerticalScrollBarVisibility]() properties respectively. By default, the visibility of both the horizontal and vertical scrollbars is set to `Default`.
+
```

Comment:
The visibility of the horizontal and vertical scrollbars can be customized using the [SfDataGrid.HorizontalScrollBarVisibility]() and [SfDataGrid.VerticalScrollBarVisibility]() properties. By default, the visibility of both the horizontal and vertical scrollbars is set to `ScrollBarVisibility.Default`.

The following code snippets demonstrate how to hide the vertical and horizontal scrollbars:

Reply:
updated